helpers/crypto/crypto.go

```diff
@@ -12,15 +12,17 @@ import (
 )
 
 var block cipher.Block
+var symmetricKey string
 
 func init() {
-	if os.Getenv("SYMMETRIC_KEY") == "" {
+	symmetricKey = os.Getenv("SYMMETRIC_KEY")
```

Comment:
I don't think this variable is helpful. We can just call `os.Getenv("SYMMETRIC_KEY")` twice.